fastlane/lib/fastlane/actions/clean_build_artifacts.rb

```diff
@@ -35,12 +35,12 @@ def self.available_options
       end
 
       def self.description
-        "Deletes files created as result of running ipa, cert, sigh or download_dsyms"
+        "Deletes files created as result of running gym, cert, sigh or download_dsyms"
```

Comment:
The description is printed in the action list also, so we don't always render it in markdown